.github/workflows/prod-build-test.yml

```
@@ -22,35 +22,61 @@ jobs:
         uses: actions/setup-node@v3
```

actionlint (1.7.7)

the runner of "actions/setup-node@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)